src/App.js

```diff
@@ -13,7 +13,15 @@ export default function App() {
     .catch(error => handleGetCharacterError(error))
 
   function createCards(characters) {
-    const cards = characters.map(character => Card(character.name))
+    const cards = characters.map(character =>
```

Comment:
You could also use de-structuring assignment here:
```suggestion
    const cards = characters.map(({ image, name, house, ancestry, patronus }) =>
```